misc/scripts/build.sh

```diff
@@ -122,7 +122,8 @@ then
 elif [ "$1" == "pylint" ]
 then
   $DOCKER run --rm -it -v $PWD/..:$dir -w $dir $pylint_container $MAKE_ESX pylint
-elif [ "$1" == "build" ] || [ "$1" == "clean-as-root" ]
+elif [ "$1" == "build" ] || [ "$1" == "clean-as-root" ] || [ "$1" == "testasroot" ]
```

Comment:
We should make this the default condition, unless a special rule exists run make using the $plug_container and the $MAKE cmd. Change this to ```else``` instead of ```elseif```